pkgs/by-name/nf/nf-core/package.nix

```diff
@@ -0,0 +1,4 @@
+{ python3Packages }:
+
+with python3Packages;
```

Comment:
There's quite a few packages that use the `with python3Packages` FYI. 

Removed it here, just in case you were feeling inspired and wanted to make a PR to fix them 😉

Reply:
> There's quite a few packages that use the `with python3Packages` FYI.

Yea I know.. Such Guides as that https://nix.dev has was written long after these bad habits were aggregated...

> Removed it here, just in case you were feeling inspired and wanted to make a PR to fix them 😉

It's not that important :) These expressions are not that long and there are not that many of them.